Flaky test TestPhoenixTypeMapping with phoenix5

`testArray` @ https://github.com/trinodb/trino/runs/2055534982 

```
2021-03-08T12:36:06.0701166Z [ERROR] testArray(io.trino.plugin.phoenix5.TestPhoenixTypeMapping)  Time elapsed: 16.354 s  <<< FAILURE!
2021-03-08T12:36:06.0702499Z java.lang.RuntimeException: 
2021-03-08T12:36:06.0703135Z Error executing sql:
2021-03-08T12:36:06.0703609Z DROP TABLE test_array_char_1xb5p
2021-03-08T12:36:06.0705067Z 	at io.trino.testing.sql.TrinoSqlExecutor.execute(TrinoSqlExecutor.java:45)
2021-03-08T12:36:06.0706637Z 	at io.trino.testing.sql.TestTable.close(TestTable.java:121)
2021-03-08T12:36:06.0708169Z 	at io.trino.testing.datatype.DataTypeTest.execute(DataTypeTest.java:95)
2021-03-08T12:36:06.0709911Z 	at io.trino.testing.datatype.DataTypeTest.execute(DataTypeTest.java:82)
2021-03-08T12:36:06.0712106Z 	at io.trino.plugin.phoenix5.TestPhoenixTypeMapping.testArray(TestPhoenixTypeMapping.java:267)
2021-03-08T12:36:06.0714484Z 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2021-03-08T12:36:06.0716826Z 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2021-03-08T12:36:06.0719574Z 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2021-03-08T12:36:06.0721605Z 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
2021-03-08T12:36:06.0723451Z 	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
2021-03-08T12:36:06.0725454Z 	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
2021-03-08T12:36:06.0726950Z 	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
2021-03-08T12:36:06.0728703Z 	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
2021-03-08T12:36:06.0730714Z 	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
2021-03-08T12:36:06.0732665Z 	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
2021-03-08T12:36:06.0734556Z 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
2021-03-08T12:36:06.0736528Z 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
2021-03-08T12:36:06.0737843Z 	at java.base/java.lang.Thread.run(Thread.java:834)
2021-03-08T12:36:06.0738803Z Caused by: java.lang.RuntimeException: Failed after attempts=4, exceptions:
2021-03-08T12:36:06.0741829Z 2021-03-08T12:35:22.937Z, RpcRetryingCaller{globalStartTime=1615206922933, pause=100, maxAttempts=4}, org.apache.hadoop.hbase.NotServingRegionException: org.apache.hadoop.hbase.NotServingRegionException: hbase:meta,,1.1588230740 is closing
2021-03-08T12:36:06.0745161Z 	at org.apache.hadoop.hbase.regionserver.HRegion.startRegionOperation(HRegion.java:8455)
2021-03-08T12:36:06.0747591Z 	at org.apache.hadoop.hbase.regionserver.HRegion.getScanner(HRegion.java:2972)
2021-03-08T12:36:06.0749685Z 	at org.apache.hadoop.hbase.regionserver.HRegion.getScanner(HRegion.java:2967)
2021-03-08T12:36:06.0751803Z 	at org.apache.hadoop.hbase.regionserver.HRegion.getScanner(HRegion.java:2961)
2021-03-08T12:36:06.0753923Z 	at org.apache.hadoop.hbase.regionserver.RSRpcServices.get(RSRpcServices.java:2642)
2021-03-08T12:36:06.0756109Z 	at org.apache.hadoop.hbase.regionserver.RSRpcServices.get(RSRpcServices.java:2569)
2021-03-08T12:36:06.0758925Z 	at org.apache.hadoop.hbase.shaded.protobuf.generated.ClientProtos$ClientService$2.callBlockingMethod(ClientProtos.java:42274)
2021-03-08T12:36:06.0760906Z 	at org.apache.hadoop.hbase.ipc.RpcServer.call(RpcServer.java:418)
2021-03-08T12:36:06.0762220Z 	at org.apache.hadoop.hbase.ipc.CallRunner.run(CallRunner.java:133)
2021-03-08T12:36:06.0763648Z 	at org.apache.hadoop.hbase.ipc.RpcExecutor$Handler.run(RpcExecutor.java:338)
2021-03-08T12:36:06.0764964Z 	at org.apache.hadoop.hbase.ipc.RpcExecutor$Handler.run(RpcExecutor.java:318)
2021-03-08T12:36:06.0765693Z 
2021-03-08T12:36:06.0767825Z 2021-03-08T12:35:23.090Z, RpcRetryingCaller{globalStartTime=1615206922933, pause=100, maxAttempts=4}, org.apache.hadoop.hbase.exceptions.RegionMovedException: Region moved to: hostname=fv-az167-365.internal.cloudapp.net port=44363 startCode=1615206605672. As of locationSeqNum=1109.
2021-03-08T12:36:06.0772023Z 2021-03-08T12:35:23.292Z, RpcRetryingCaller{globalStartTime=1615206922933, pause=100, maxAttempts=4}, org.apache.hadoop.hbase.NotServingRegionException: org.apache.hadoop.hbase.NotServingRegionException: hbase:meta,,1 is not online on fv-az167-365.internal.cloudapp.net,44363,1615206605672
2021-03-08T12:36:06.0775616Z 	at org.apache.hadoop.hbase.regionserver.HRegionServer.getRegionByEncodedName(HRegionServer.java:3382)
2021-03-08T12:36:06.0778209Z 	at org.apache.hadoop.hbase.regionserver.HRegionServer.getRegion(HRegionServer.java:3359)
2021-03-08T12:36:06.0780380Z 	at org.apache.hadoop.hbase.regionserver.RSRpcServices.getRegion(RSRpcServices.java:1467)
2021-03-08T12:36:06.0782423Z 	at org.apache.hadoop.hbase.regionserver.RSRpcServices.get(RSRpcServices.java:2545)
2021-03-08T12:36:06.0784682Z 	at org.apache.hadoop.hbase.shaded.protobuf.generated.ClientProtos$ClientService$2.callBlockingMethod(ClientProtos.java:42274)
2021-03-08T12:36:06.0786624Z 	at org.apache.hadoop.hbase.ipc.RpcServer.call(RpcServer.java:418)
2021-03-08T12:36:06.0787940Z 	at org.apache.hadoop.hbase.ipc.CallRunner.run(CallRunner.java:133)
2021-03-08T12:36:06.0789252Z 	at org.apache.hadoop.hbase.ipc.RpcExecutor$Handler.run(RpcExecutor.java:338)
2021-03-08T12:36:06.0790577Z 	at org.apache.hadoop.hbase.ipc.RpcExecutor$Handler.run(RpcExecutor.java:318)
2021-03-08T12:36:06.0791302Z 
2021-03-08T12:36:06.0793400Z 2021-03-08T12:35:23.593Z, RpcRetryingCaller{globalStartTime=1615206922933, pause=100, maxAttempts=4}, org.apache.hadoop.hbase.exceptions.RegionMovedException: Region moved to: hostname=fv-az167-365.internal.cloudapp.net port=44363 startCode=1615206605672. As of locationSeqNum=1109.
2021-03-08T12:36:06.0795060Z 
2021-03-08T12:36:06.0796245Z 	at io.trino.testing.AbstractTestingTrinoClient.execute(AbstractTestingTrinoClient.java:120)
2021-03-08T12:36:06.0798239Z 	at io.trino.testing.DistributedQueryRunner.execute(DistributedQueryRunner.java:491)
2021-03-08T12:36:06.0799930Z 	at io.trino.testing.sql.TrinoSqlExecutor.execute(TrinoSqlExecutor.java:42)
2021-03-08T12:36:06.0800856Z 	... 17 more
2021-03-08T12:36:06.0801508Z Caused by: io.trino.spi.TrinoException: Failed after attempts=4, exceptions:
2021-03-08T12:36:06.0804005Z 2021-03-08T12:35:22.937Z, RpcRetryingCaller{globalStartTime=1615206922933, pause=100, maxAttempts=4}, org.apache.hadoop.hbase.NotServingRegionException: org.apache.hadoop.hbase.NotServingRegionException: hbase:meta,,1.1588230740 is closing
2021-03-08T12:36:06.0806894Z 	at org.apache.hadoop.hbase.regionserver.HRegion.startRegionOperation(HRegion.java:8455)
2021-03-08T12:36:06.0809050Z 	at org.apache.hadoop.hbase.regionserver.HRegion.getScanner(HRegion.java:2972)
2021-03-08T12:36:06.0810868Z 	at org.apache.hadoop.hbase.regionserver.HRegion.getScanner(HRegion.java:2967)
2021-03-08T12:36:06.0812666Z 	at org.apache.hadoop.hbase.regionserver.HRegion.getScanner(HRegion.java:2961)
2021-03-08T12:36:06.0814521Z 	at org.apache.hadoop.hbase.regionserver.RSRpcServices.get(RSRpcServices.java:2642)
2021-03-08T12:36:06.0816402Z 	at org.apache.hadoop.hbase.regionserver.RSRpcServices.get(RSRpcServices.java:2569)
2021-03-08T12:36:06.0818746Z 	at org.apache.hadoop.hbase.shaded.protobuf.generated.ClientProtos$ClientService$2.callBlockingMethod(ClientProtos.java:42274)
2021-03-08T12:36:06.0820700Z 	at org.apache.hadoop.hbase.ipc.RpcServer.call(RpcServer.java:418)
2021-03-08T12:36:06.0822019Z 	at org.apache.hadoop.hbase.ipc.CallRunner.run(CallRunner.java:133)
2021-03-08T12:36:06.0823382Z 	at org.apache.hadoop.hbase.ipc.RpcExecutor$Handler.run(RpcExecutor.java:338)
2021-03-08T12:36:06.0824706Z 	at org.apache.hadoop.hbase.ipc.RpcExecutor$Handler.run(RpcExecutor.java:318)
2021-03-08T12:36:06.0825419Z 
2021-03-08T12:36:06.0827526Z 2021-03-08T12:35:23.090Z, RpcRetryingCaller{globalStartTime=1615206922933, pause=100, maxAttempts=4}, org.apache.hadoop.hbase.exceptions.RegionMovedException: Region moved to: hostname=fv-az167-365.internal.cloudapp.net port=44363 startCode=1615206605672. As of locationSeqNum=1109.
2021-03-08T12:36:06.0831586Z 2021-03-08T12:35:23.292Z, RpcRetryingCaller{globalStartTime=1615206922933, pause=100, maxAttempts=4}, org.apache.hadoop.hbase.NotServingRegionException: org.apache.hadoop.hbase.NotServingRegionException: hbase:meta,,1 is not online on fv-az167-365.internal.cloudapp.net,44363,1615206605672
2021-03-08T12:36:06.0835212Z 	at org.apache.hadoop.hbase.regionserver.HRegionServer.getRegionByEncodedName(HRegionServer.java:3382)
2021-03-08T12:36:06.0837908Z 	at org.apache.hadoop.hbase.regionserver.HRegionServer.getRegion(HRegionServer.java:3359)
2021-03-08T12:36:06.0840109Z 	at org.apache.hadoop.hbase.regionserver.RSRpcServices.getRegion(RSRpcServices.java:1467)
2021-03-08T12:36:06.0842135Z 	at org.apache.hadoop.hbase.regionserver.RSRpcServices.get(RSRpcServices.java:2545)
2021-03-08T12:36:06.0844403Z 	at org.apache.hadoop.hbase.shaded.protobuf.generated.ClientProtos$ClientService$2.callBlockingMethod(ClientProtos.java:42274)
2021-03-08T12:36:06.0846340Z 	at org.apache.hadoop.hbase.ipc.RpcServer.call(RpcServer.java:418)
2021-03-08T12:36:06.0847656Z 	at org.apache.hadoop.hbase.ipc.CallRunner.run(CallRunner.java:133)
2021-03-08T12:36:06.0849151Z 	at org.apache.hadoop.hbase.ipc.RpcExecutor$Handler.run(RpcExecutor.java:338)
2021-03-08T12:36:06.0850480Z 	at org.apache.hadoop.hbase.ipc.RpcExecutor$Handler.run(RpcExecutor.java:318)
2021-03-08T12:36:06.0851202Z 
2021-03-08T12:36:06.0853307Z 2021-03-08T12:35:23.593Z, RpcRetryingCaller{globalStartTime=1615206922933, pause=100, maxAttempts=4}, org.apache.hadoop.hbase.exceptions.RegionMovedException: Region moved to: hostname=fv-az167-365.internal.cloudapp.net port=44363 startCode=1615206605672. As of locationSeqNum=1109.
2021-03-08T12:36:06.0854978Z 
2021-03-08T12:36:06.0855827Z 	at io.trino.plugin.jdbc.BaseJdbcClient.execute(BaseJdbcClient.java:976)
2021-03-08T12:36:06.0857270Z 	at io.trino.plugin.jdbc.BaseJdbcClient.execute(BaseJdbcClient.java:962)
2021-03-08T12:36:06.0858754Z 	at io.trino.plugin.phoenix5.PhoenixClient.execute(PhoenixClient.java:220)
2021-03-08T12:36:06.0860279Z 	at io.trino.plugin.jdbc.BaseJdbcClient.dropTable(BaseJdbcClient.java:776)
2021-03-08T12:36:06.0861904Z 	at io.trino.plugin.phoenix5.PhoenixMetadata.dropTable(PhoenixMetadata.java:220)
2021-03-08T12:36:06.0864604Z 	at io.trino.plugin.base.classloader.ClassLoaderSafeConnectorMetadata.dropTable(ClassLoaderSafeConnectorMetadata.java:345)
2021-03-08T12:36:06.0867147Z 	at io.trino.metadata.MetadataManager.dropTable(MetadataManager.java:746)
2021-03-08T12:36:06.0868563Z 	at io.trino.execution.DropTableTask.execute(DropTableTask.java:67)
2021-03-08T12:36:06.0869887Z 	at io.trino.execution.DropTableTask.execute(DropTableTask.java:35)
2021-03-08T12:36:06.0871493Z 	at io.trino.execution.DataDefinitionExecution.start(DataDefinitionExecution.java:170)
2021-03-08T12:36:06.0873233Z 	at io.trino.execution.SqlQueryManager.createQuery(SqlQueryManager.java:237)
2021-03-08T12:36:06.0874937Z 	at io.trino.dispatcher.LocalDispatchQuery.lambda$startExecution$7(LocalDispatchQuery.java:143)
2021-03-08T12:36:06.0876159Z 	at io.trino.$gen.Trino_testversion____20210308_123503_905.run(Unknown Source)
2021-03-08T12:36:06.0877430Z 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
2021-03-08T12:36:06.0879255Z 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
2021-03-08T12:36:06.0880373Z 	at java.base/java.lang.Thread.run(Thread.java:834)
2021-03-08T12:36:06.0881308Z 	Suppressed: java.lang.RuntimeException: Query: DROP TABLE "TPCH"."TEST_ARRAY_CHAR_1XB5P"
2021-03-08T12:36:06.0882542Z 		at io.trino.plugin.jdbc.BaseJdbcClient.execute(BaseJdbcClient.java:977)
2021-03-08T12:36:06.0883420Z 		... 15 more
2021-03-08T12:36:06.0884427Z Caused by: org.apache.phoenix.exception.PhoenixIOException: Failed after attempts=4, exceptions:
2021-03-08T12:36:06.0887327Z 2021-03-08T12:35:22.937Z, RpcRetryingCaller{globalStartTime=1615206922933, pause=100, maxAttempts=4}, org.apache.hadoop.hbase.NotServingRegionException: org.apache.hadoop.hbase.NotServingRegionException: hbase:meta,,1.1588230740 is closing
2021-03-08T12:36:06.0890305Z 	at org.apache.hadoop.hbase.regionserver.HRegion.startRegionOperation(HRegion.java:8455)
2021-03-08T12:36:06.0892416Z 	at org.apache.hadoop.hbase.regionserver.HRegion.getScanner(HRegion.java:2972)
2021-03-08T12:36:06.0894221Z 	at org.apache.hadoop.hbase.regionserver.HRegion.getScanner(HRegion.java:2967)
2021-03-08T12:36:06.0896050Z 	at org.apache.hadoop.hbase.regionserver.HRegion.getScanner(HRegion.java:2961)
2021-03-08T12:36:06.0897886Z 	at org.apache.hadoop.hbase.regionserver.RSRpcServices.get(RSRpcServices.java:2642)
2021-03-08T12:36:06.0899774Z 	at org.apache.hadoop.hbase.regionserver.RSRpcServices.get(RSRpcServices.java:2569)
2021-03-08T12:36:06.0902009Z 	at org.apache.hadoop.hbase.shaded.protobuf.generated.ClientProtos$ClientService$2.callBlockingMethod(ClientProtos.java:42274)
2021-03-08T12:36:06.0903991Z 	at org.apache.hadoop.hbase.ipc.RpcServer.call(RpcServer.java:418)
2021-03-08T12:36:06.0905295Z 	at org.apache.hadoop.hbase.ipc.CallRunner.run(CallRunner.java:133)
2021-03-08T12:36:06.0906617Z 	at org.apache.hadoop.hbase.ipc.RpcExecutor$Handler.run(RpcExecutor.java:338)
2021-03-08T12:36:06.0907972Z 	at org.apache.hadoop.hbase.ipc.RpcExecutor$Handler.run(RpcExecutor.java:318)
2021-03-08T12:36:06.0908698Z 
2021-03-08T12:36:06.0910788Z 2021-03-08T12:35:23.090Z, RpcRetryingCaller{globalStartTime=1615206922933, pause=100, maxAttempts=4}, org.apache.hadoop.hbase.exceptions.RegionMovedException: Region moved to: hostname=fv-az167-365.internal.cloudapp.net port=44363 startCode=1615206605672. As of locationSeqNum=1109.
2021-03-08T12:36:06.0914876Z 2021-03-08T12:35:23.292Z, RpcRetryingCaller{globalStartTime=1615206922933, pause=100, maxAttempts=4}, org.apache.hadoop.hbase.NotServingRegionException: org.apache.hadoop.hbase.NotServingRegionException: hbase:meta,,1 is not online on fv-az167-365.internal.cloudapp.net,44363,1615206605672
2021-03-08T12:36:06.0918496Z 	at org.apache.hadoop.hbase.regionserver.HRegionServer.getRegionByEncodedName(HRegionServer.java:3382)
2021-03-08T12:36:06.0921081Z 	at org.apache.hadoop.hbase.regionserver.HRegionServer.getRegion(HRegionServer.java:3359)
2021-03-08T12:36:06.0923286Z 	at org.apache.hadoop.hbase.regionserver.RSRpcServices.getRegion(RSRpcServices.java:1467)
2021-03-08T12:36:06.0925306Z 	at org.apache.hadoop.hbase.regionserver.RSRpcServices.get(RSRpcServices.java:2545)
2021-03-08T12:36:06.0927552Z 	at org.apache.hadoop.hbase.shaded.protobuf.generated.ClientProtos$ClientService$2.callBlockingMethod(ClientProtos.java:42274)
2021-03-08T12:36:06.0929625Z 	at org.apache.hadoop.hbase.ipc.RpcServer.call(RpcServer.java:418)
2021-03-08T12:36:06.0930927Z 	at org.apache.hadoop.hbase.ipc.CallRunner.run(CallRunner.java:133)
2021-03-08T12:36:06.0932254Z 	at org.apache.hadoop.hbase.ipc.RpcExecutor$Handler.run(RpcExecutor.java:338)
2021-03-08T12:36:06.0933571Z 	at org.apache.hadoop.hbase.ipc.RpcExecutor$Handler.run(RpcExecutor.java:318)
2021-03-08T12:36:06.0934295Z 
2021-03-08T12:36:06.0936396Z 2021-03-08T12:35:23.593Z, RpcRetryingCaller{globalStartTime=1615206922933, pause=100, maxAttempts=4}, org.apache.hadoop.hbase.exceptions.RegionMovedException: Region moved to: hostname=fv-az167-365.internal.cloudapp.net port=44363 startCode=1615206605672. As of locationSeqNum=1109.
2021-03-08T12:36:06.0938164Z 
2021-03-08T12:36:06.0939260Z 	at org.apache.phoenix.util.ServerUtil.parseServerException(ServerUtil.java:138)
2021-03-08T12:36:06.0941676Z 	at org.apache.phoenix.query.ConnectionQueryServicesImpl.dropTables(ConnectionQueryServicesImpl.java:2182)
2021-03-08T12:36:06.0944464Z 	at org.apache.phoenix.query.ConnectionQueryServicesImpl.dropTable(ConnectionQueryServicesImpl.java:2070)
2021-03-08T12:36:06.0946693Z 	at org.apache.phoenix.schema.MetaDataClient.dropTable(MetaDataClient.java:3431)
2021-03-08T12:36:06.0948438Z 	at org.apache.phoenix.schema.MetaDataClient.dropTable(MetaDataClient.java:3339)
2021-03-08T12:36:06.0950264Z 	at org.apache.phoenix.jdbc.PhoenixStatement$ExecutableDropTableStatement$1.execute(PhoenixStatement.java:1204)
2021-03-08T12:36:06.0951957Z 	at org.apache.phoenix.jdbc.PhoenixStatement$2.call(PhoenixStatement.java:415)
2021-03-08T12:36:06.0953363Z 	at org.apache.phoenix.jdbc.PhoenixStatement$2.call(PhoenixStatement.java:397)
2021-03-08T12:36:06.0954665Z 	at org.apache.phoenix.call.CallRunner.run(CallRunner.java:53)
2021-03-08T12:36:06.0956269Z 	at org.apache.phoenix.jdbc.PhoenixStatement.executeMutation(PhoenixStatement.java:396)
2021-03-08T12:36:06.0958344Z 	at org.apache.phoenix.jdbc.PhoenixStatement.executeMutation(PhoenixStatement.java:384)
2021-03-08T12:36:06.0960246Z 	at org.apache.phoenix.jdbc.PhoenixStatement.execute(PhoenixStatement.java:1925)
2021-03-08T12:36:06.0961811Z 	at io.trino.plugin.jdbc.BaseJdbcClient.execute(BaseJdbcClient.java:973)
2021-03-08T12:36:06.0962688Z 	... 15 more
2021-03-08T12:36:06.0963875Z Caused by: org.apache.hadoop.hbase.client.RetriesExhaustedException: Failed after attempts=4, exceptions:
2021-03-08T12:36:06.0966938Z 2021-03-08T12:35:22.937Z, RpcRetryingCaller{globalStartTime=1615206922933, pause=100, maxAttempts=4}, org.apache.hadoop.hbase.NotServingRegionException: org.apache.hadoop.hbase.NotServingRegionException: hbase:meta,,1.1588230740 is closing
2021-03-08T12:36:06.0969888Z 	at org.apache.hadoop.hbase.regionserver.HRegion.startRegionOperation(HRegion.java:8455)
2021-03-08T12:36:06.0971973Z 	at org.apache.hadoop.hbase.regionserver.HRegion.getScanner(HRegion.java:2972)
2021-03-08T12:36:06.0973785Z 	at org.apache.hadoop.hbase.regionserver.HRegion.getScanner(HRegion.java:2967)
2021-03-08T12:36:06.0975605Z 	at org.apache.hadoop.hbase.regionserver.HRegion.getScanner(HRegion.java:2961)
2021-03-08T12:36:06.0977438Z 	at org.apache.hadoop.hbase.regionserver.RSRpcServices.get(RSRpcServices.java:2642)
2021-03-08T12:36:06.0979314Z 	at org.apache.hadoop.hbase.regionserver.RSRpcServices.get(RSRpcServices.java:2569)
2021-03-08T12:36:06.0981556Z 	at org.apache.hadoop.hbase.shaded.protobuf.generated.ClientProtos$ClientService$2.callBlockingMethod(ClientProtos.java:42274)
2021-03-08T12:36:06.0983526Z 	at org.apache.hadoop.hbase.ipc.RpcServer.call(RpcServer.java:418)
2021-03-08T12:36:06.0984834Z 	at org.apache.hadoop.hbase.ipc.CallRunner.run(CallRunner.java:133)
2021-03-08T12:36:06.0986175Z 	at org.apache.hadoop.hbase.ipc.RpcExecutor$Handler.run(RpcExecutor.java:338)
2021-03-08T12:36:06.0987500Z 	at org.apache.hadoop.hbase.ipc.RpcExecutor$Handler.run(RpcExecutor.java:318)
2021-03-08T12:36:06.0988251Z 
2021-03-08T12:36:06.0990369Z 2021-03-08T12:35:23.090Z, RpcRetryingCaller{globalStartTime=1615206922933, pause=100, maxAttempts=4}, org.apache.hadoop.hbase.exceptions.RegionMovedException: Region moved to: hostname=fv-az167-365.internal.cloudapp.net port=44363 startCode=1615206605672. As of locationSeqNum=1109.
2021-03-08T12:36:06.0994475Z 2021-03-08T12:35:23.292Z, RpcRetryingCaller{globalStartTime=1615206922933, pause=100, maxAttempts=4}, org.apache.hadoop.hbase.NotServingRegionException: org.apache.hadoop.hbase.NotServingRegionException: hbase:meta,,1 is not online on fv-az167-365.internal.cloudapp.net,44363,1615206605672
2021-03-08T12:36:06.0998088Z 	at org.apache.hadoop.hbase.regionserver.HRegionServer.getRegionByEncodedName(HRegionServer.java:3382)
2021-03-08T12:36:06.1000797Z 	at org.apache.hadoop.hbase.regionserver.HRegionServer.getRegion(HRegionServer.java:3359)
2021-03-08T12:36:06.1002977Z 	at org.apache.hadoop.hbase.regionserver.RSRpcServices.getRegion(RSRpcServices.java:1467)
2021-03-08T12:36:06.1005088Z 	at org.apache.hadoop.hbase.regionserver.RSRpcServices.get(RSRpcServices.java:2545)
2021-03-08T12:36:06.1007374Z 	at org.apache.hadoop.hbase.shaded.protobuf.generated.ClientProtos$ClientService$2.callBlockingMethod(ClientProtos.java:42274)
2021-03-08T12:36:06.1009418Z 	at org.apache.hadoop.hbase.ipc.RpcServer.call(RpcServer.java:418)
2021-03-08T12:36:06.1010728Z 	at org.apache.hadoop.hbase.ipc.CallRunner.run(CallRunner.java:133)
2021-03-08T12:36:06.1012050Z 	at org.apache.hadoop.hbase.ipc.RpcExecutor$Handler.run(RpcExecutor.java:338)
2021-03-08T12:36:06.1013368Z 	at org.apache.hadoop.hbase.ipc.RpcExecutor$Handler.run(RpcExecutor.java:318)
2021-03-08T12:36:06.1014086Z 
2021-03-08T12:36:06.1016202Z 2021-03-08T12:35:23.593Z, RpcRetryingCaller{globalStartTime=1615206922933, pause=100, maxAttempts=4}, org.apache.hadoop.hbase.exceptions.RegionMovedException: Region moved to: hostname=fv-az167-365.internal.cloudapp.net port=44363 startCode=1615206605672. As of locationSeqNum=1109.
2021-03-08T12:36:06.1017865Z 
2021-03-08T12:36:06.1019445Z 	at org.apache.hadoop.hbase.client.RpcRetryingCallerImpl.callWithRetries(RpcRetryingCallerImpl.java:145)
2021-03-08T12:36:06.1021475Z 	at org.apache.hadoop.hbase.client.HTable.get(HTable.java:384)
2021-03-08T12:36:06.1022670Z 	at org.apache.hadoop.hbase.client.HTable.get(HTable.java:358)
2021-03-08T12:36:06.1024303Z 	at org.apache.hadoop.hbase.MetaTableAccessor.getTableState(MetaTableAccessor.java:1118)
2021-03-08T12:36:06.1026328Z 	at org.apache.hadoop.hbase.MetaTableAccessor.tableExists(MetaTableAccessor.java:440)
2021-03-08T12:36:06.1029005Z 	at org.apache.hadoop.hbase.master.procedure.DisableTableProcedure.prepareDisable(DisableTableProcedure.java:248)
2021-03-08T12:36:06.1032512Z 	at org.apache.hadoop.hbase.master.procedure.DisableTableProcedure.executeFromState(DisableTableProcedure.java:91)
2021-03-08T12:36:06.1036096Z 	at org.apache.hadoop.hbase.master.procedure.DisableTableProcedure.executeFromState(DisableTableProcedure.java:46)
2021-03-08T12:36:06.1039180Z 	at org.apache.hadoop.hbase.procedure2.StateMachineProcedure.execute(StateMachineProcedure.java:194)
2021-03-08T12:36:06.1041401Z 	at org.apache.hadoop.hbase.procedure2.Procedure.doExecute(Procedure.java:962)
2021-03-08T12:36:06.1043613Z 	at org.apache.hadoop.hbase.procedure2.ProcedureExecutor.execProcedure(ProcedureExecutor.java:1662)
2021-03-08T12:36:06.1046316Z 	at org.apache.hadoop.hbase.procedure2.ProcedureExecutor.executeProcedure(ProcedureExecutor.java:1409)
2021-03-08T12:36:06.1049004Z 	at org.apache.hadoop.hbase.procedure2.ProcedureExecutor.access$1100(ProcedureExecutor.java:78)
2021-03-08T12:36:06.1051154Z 	at org.apache.hadoop.hbase.procedure2.ProcedureExecutor$WorkerThread.run(ProcedureExecutor.java:1979)
2021-03-08T12:36:06.1052264Z 
2021-03-08T12:36:06.1052927Z 2021-03-08T06:36:05.500-0600 INFO Life cycle stopping...
2021-03-08T12:36:06.1053568Z 2021-03-08T06:36:05.501-0600 INFO Life cycle stopped
2021-03-08T12:36:06.1054290Z 2021-03-08T06:36:05.502-0600 INFO Stopped http@45920cef{HTTP/1.1, (http/1.1, h2c)}{127.0.0.1:41915}
2021-03-08T12:36:06.1055341Z 2021-03-08T06:36:05.516-0600 INFO Stopped o.e.j.s.ServletContextHandler@7a854a5b{/,null,UNAVAILABLE,@http}
2021-03-08T12:36:06.1056756Z 2021-03-08T06:36:05.517-0600 WARNING QueuedThreadPool[http-worker]@38024040{STOPPING,2<=0<=200,i=9,r=-1,q=0}[NO_TRY] Couldn't stop Thread[http-worker-9349,5,main]
2021-03-08T12:36:06.1058251Z 2021-03-08T06:36:05.517-0600 WARNING QueuedThreadPool[http-worker]@38024040{STOPPING,2<=0<=200,i=9,r=-1,q=0}[NO_TRY] Couldn't stop Thread[http-worker-9352,5,main]
2021-03-08T12:36:06.1059720Z 2021-03-08T06:36:05.517-0600 WARNING QueuedThreadPool[http-worker]@38024040{STOPPING,2<=0<=200,i=9,r=-1,q=0}[NO_TRY] Couldn't stop Thread[http-worker-9544,5,main]
2021-03-08T12:36:06.1061299Z 2021-03-08T06:36:05.517-0600 WARNING QueuedThreadPool[http-worker]@38024040{STOPPING,2<=0<=200,i=9,r=-1,q=0}[NO_TRY] Couldn't stop Thread[http-worker-9351,5,]
2021-03-08T12:36:06.1062765Z 2021-03-08T06:36:05.517-0600 WARNING QueuedThreadPool[http-worker]@38024040{STOPPING,2<=0<=200,i=9,r=-1,q=0}[NO_TRY] Couldn't stop Thread[http-worker-10085,5,]
2021-03-08T12:36:06.1064191Z 2021-03-08T06:36:05.517-0600 WARNING QueuedThreadPool[http-worker]@38024040{STOPPING,2<=0<=200,i=9,r=-1,q=0}[NO_TRY] Couldn't stop Thread[http-worker-10844,5,]
2021-03-08T12:36:06.1065620Z 2021-03-08T06:36:05.517-0600 WARNING QueuedThreadPool[http-worker]@38024040{STOPPING,2<=0<=200,i=9,r=-1,q=0}[NO_TRY] Couldn't stop Thread[http-worker-9560,5,main]
2021-03-08T12:36:06.1066608Z 2021-03-08T06:36:05.531-0600 INFO Life cycle stopped
2021-03-08T12:36:06.1067235Z 2021-03-08T06:36:05.531-0600 INFO Life cycle stopping...
2021-03-08T12:36:06.1067977Z 2021-03-08T06:36:05.532-0600 INFO Stopped http@2599a780{HTTP/1.1, (http/1.1, h2c)}{127.0.0.1:33505}
2021-03-08T12:36:06.1069012Z 2021-03-08T06:36:05.534-0600 INFO Stopped o.e.j.s.ServletContextHandler@4ca2f683{/,null,UNAVAILABLE,@http}
2021-03-08T12:36:06.1070439Z 2021-03-08T06:36:05.535-0600 WARNING QueuedThreadPool[http-worker]@787d35c4{STOPPING,2<=0<=200,i=5,r=-1,q=0}[NO_TRY] Couldn't stop Thread[http-worker-9362,5,main]
2021-03-08T12:36:06.1071934Z 2021-03-08T06:36:05.535-0600 WARNING QueuedThreadPool[http-worker]@787d35c4{STOPPING,2<=0<=200,i=5,r=-1,q=0}[NO_TRY] Couldn't stop Thread[http-worker-9329,5,main]
2021-03-08T12:36:06.1073438Z 2021-03-08T06:36:05.535-0600 WARNING QueuedThreadPool[http-worker]@787d35c4{STOPPING,2<=0<=200,i=5,r=-1,q=0}[NO_TRY] Couldn't stop Thread[http-worker-9320,5,main]
2021-03-08T12:36:06.1074920Z 2021-03-08T06:36:05.535-0600 WARNING QueuedThreadPool[http-worker]@787d35c4{STOPPING,2<=0<=200,i=5,r=-1,q=0}[NO_TRY] Couldn't stop Thread[http-worker-11271,5,main]
2021-03-08T12:36:06.1076419Z 2021-03-08T06:36:05.535-0600 WARNING QueuedThreadPool[http-worker]@787d35c4{STOPPING,2<=0<=200,i=5,r=-1,q=0}[NO_TRY] Couldn't stop Thread[http-worker-11270,5,main]
2021-03-08T12:36:06.1077419Z 2021-03-08T06:36:05.538-0600 INFO Life cycle stopped
2021-03-08T12:36:06.2557830Z [INFO] 
2021-03-08T12:36:06.2558755Z [INFO] Results:
2021-03-08T12:36:06.2559171Z [INFO] 
2021-03-08T12:36:06.2559569Z [ERROR] Failures: 
2021-03-08T12:36:06.2572691Z [ERROR]   TestPhoenixTypeMapping.testArray:267 » Runtime Error executing sql:
2021-03-08T12:36:06.2573697Z DROP TABLE...
```

cc @lhofhansl

--- a/plugin/trino-phoenix/src/test/java/io/trino/plugin/phoenix/TestingPhoenixServer.java
+++ b/plugin/trino-phoenix/src/test/java/io/trino/plugin/phoenix/TestingPhoenixServer.java
@@ -89,7 +89,7 @@ public final class TestingPhoenixServer
         this.conf.set("hbase.security.logger", "ERROR");
         this.conf.setInt(MASTER_INFO_PORT, -1);
         this.conf.setInt(REGIONSERVER_INFO_PORT, -1);
-        this.conf.setInt(HBASE_CLIENT_RETRIES_NUMBER, 1);
+        this.conf.setInt(HBASE_CLIENT_RETRIES_NUMBER, 15);
         this.conf.setBoolean("phoenix.schema.isNamespaceMappingEnabled", true);
         this.conf.set("hbase.regionserver.wal.codec", "org.apache.hadoop.hbase.regionserver.wal.IndexedWALEditCodec");
         this.hbaseTestingUtility = new HBaseTestingUtility(conf);

--- a/plugin/trino-phoenix5/src/test/java/io/trino/plugin/phoenix5/TestingPhoenixServer.java
+++ b/plugin/trino-phoenix5/src/test/java/io/trino/plugin/phoenix5/TestingPhoenixServer.java
@@ -89,7 +89,7 @@ public final class TestingPhoenixServer
         this.conf.set("hbase.security.logger", "ERROR");
         this.conf.setInt(MASTER_INFO_PORT, -1);
         this.conf.setInt(REGIONSERVER_INFO_PORT, -1);
-        this.conf.setInt(HBASE_CLIENT_RETRIES_NUMBER, 1);
+        this.conf.setInt(HBASE_CLIENT_RETRIES_NUMBER, 15);
         this.conf.setBoolean("phoenix.schema.isNamespaceMappingEnabled", true);
         this.conf.set("hbase.regionserver.wal.codec", "org.apache.hadoop.hbase.regionserver.wal.IndexedWALEditCodec");
         this.hbaseTestingUtility = new HBaseTestingUtility(conf);

